recipes/fortnet/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "fortnet" %}
+{% set version = "0.7.1" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+{% set origin = "vanderhe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ origin }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a7537585e947e3352324b866533b7756a6cfb5ff1524e3a2ff16929d9283cb39
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  {% if mpi != "nompi" %}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
```

```suggestion
  run_exports:
    - {{ name|lower }} * {{ mpi_prefix }}_*
    - {{ pin_subpackage( name|lower, max_pin='x.x.x') }}
```
Anyone building against this library needs the version pinned also. The max_pin of the API should be set according to ABI guarantees. MAJOR version less than 1 usually means no ABI guarantees, so pin to the PATCH level.

---

You're right, I've applied your suggestions.